versioned_docs/version-2.0/how_to_guides/datasets/index_datasets_for_dynamic_few_shot_example_selection.mdx

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 5
+---
+
+import {
+  CodeTabs,
+  PythonBlock,
+  TypeScriptBlock,
+} from "@site/src/components/InstructionsWithCode";
+
+# Dynamic few shot example selection
+
+:::note
+This feature is currently in closed beta. Please sign up [here](https://forms.gle/in9R6t9HNSYMBt7P7) for access
+:::
+
+Configure your datasets so that you can search for few shot examples based on an incoming request.
```

Comment:
Do you want to add more background here on why folks would want to use this and what the feature does?

Reply:
IIUC how to guides are not supposed to do this:

https://diataxis.fr/how-to-guides/#key-principles

> Anything else that’s added distracts both you and the user and dilutes the useful power of the guide. Typically, the temptations are to explain or to provide reference for completeness. Neither of these are part of guiding the user in their work. They get in the way of the action; if they’re important, link to them.

and

>How-to characteristics

> * focused on tasks or problems

> * assume the user knows what they want to achieve

> * action and only action

> * no digression, explanation, teaching

Reply:
We may want a tutorial on few shot prompting that goes through some teaching in addition?

Reply:
Yeah, that makes sense on tutorial